### PR TITLE
Create Helpers to aid creation of QueueManagers for each environment

### PR DESCRIPTION
**Pull request by cbeland**
We shouldn't pollute dbor_utils with connection info to NoodleQueue but rather let the QueueManager class handle this (eventually) securely.

### CODE REVIEW

**Comment by cbeland** [comment]
I don't think any of the manager stuff is actually being used anywhere, so I'm removing it.

**Comment by cbeland** [comment]
@jbradberry: This is ready for code review.

File: dborutils/batch_manager.py [hunk 1]
```diff
@@ -1,130 +0,0 @@
-from datetime import datetime
-from decimal import Decimal
-import logging
-logger = logging.getLogger(__name__)
-
-
-class BatchManager(object):
```
Comment (jbradberry) [inline dborutils/batch_manager.py]: This class still is imported in dborutils/copy_service.py, and is used there.  However, that module contains only four classes: `NoodleCopyService`, `NoodleMongoProdCopyService`, `NoodleScrapedMongoCopyService`, and `NoodleScrapedMongoListCopyService`, which are not used anywhere else.  Let's drop that module as well.

**Comment by jbradberry** [comment]
Reviewed to e56e340ba8a214192986ee12e089956d098ccd5f.  cc @cbeland

**Pull request merged**